app/build.gradle

```diff
@@ -22,12 +23,18 @@ android {
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation 'com.android.support:appcompat-v7:25.4.0'
+    implementation 'com.android.support:appcompat-v7:28.0.0'
     implementation 'org.apache.commons:commons-lang3:3.7'
     implementation 'joda-time:joda-time:2.10'
     testImplementation 'junit:junit:4.12'
     testImplementation 'org.hamcrest:hamcrest-all:1.3'
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+
+    //Test dependencies
```

Comment:
This comment is redundant and somewhat incorrect since there are other test dependencies declared just above.